### PR TITLE
LocalDataDriver for using CSV and pickled DF files as a QueryProvider

### PR DESCRIPTION
**Pull request by ianhelle**
Removed deprecated kql.py, query_builtin_queries, query_mgr.py, query_schema.py
Changed location of query_defns.py and made pkg reference updates in several modules and notebooks.
Some fixes to support local_data_driver in query_store.py, driver_base.py and data_providers.py
Unit test - test_localdata_queries.yaml and supporting data and query files.
Fixed test in test_utils.py to work on Linux
Add documentation for LocalDataDriver to DataProviders.rst and updated section on creating query files.
Reduced warnings produced during pytest run to something more reasonable.

### CODE REVIEW

File: docs/source/msticpy.nbtools.rst [hunk 1]
```diff
@@ -20,6 +20,14 @@ msticpy.nbtools.morph_charts module
     :undoc-members:
     :show-inheritance:
 
+msticpy.nbtools.nbinit module
+--------------------------------
```
Comment (petebryan) [inline docs/source/msticpy.nbtools.rst:24]: too many -

File: msticpy/sectools/tiproviders/kql_base.py [hunk 2]
```diff
@@ -208,14 +208,14 @@ def lookup_iocs(  # noqa: C901, MC0001
                     and data_result.completion_query_info["StatusCode"] == 0
                     and data_result.records_count == 0
                 ):
-                    warnings.warn("No results return from data provider.")
+                    print("No results return from data provider.")
```
Comment (petebryan) [inline msticpy/sectools/tiproviders/kql_base.py:211]: This message reads strangely, should it be "No results returned from data provider"?

File: docs/source/data_acquisition/DataProviders.rst [hunk 3]
```diff
@@ -266,6 +280,118 @@ be found `here <https://docs.microsoft.com/en-us/graph/auth-register-app-v2?cont
         mdatp_prov.connect(app_name="SecurityGraphApp")
 
 
+Local Data Provider
+-------------------
+
+:py:mod:`Security Graph driver API documentation<msticpy.data.drivers.local_data_driver>`
+
+The Local data provider is intended primarily for testing or demonstrations
```
[the remark this answers is not in the document]
Comment (ianhelle) [reply docs/source/data_acquisition/DataProviders.rst:288]: Fixed (I think)

File: docs/source/data_acquisition/DataProviders.rst [hunk 4]
```diff
@@ -266,6 +280,118 @@ be found `here <https://docs.microsoft.com/en-us/graph/auth-register-app-v2?cont
...
+or a CSV file. In either case the data is converted to a DataFrame to be returned
+from the query. Usage of this driver is a little different to most other drivers:
+
+* You will usually need to provide a path to your data files when initializing
+  the query provider.
+* You will need to provide a query definition file that maps file names on to
+  the query name.
+* Parameters to queries are ignored.
+
+An example of a LocalData yaml query file.
```
[the remark this answers is not in the document]
Comment (ianhelle) [reply docs/source/data_acquisition/DataProviders.rst:301]: Fixed

**Pull request merged**